server.h

```diff
@@ -59,7 +58,7 @@ class server  {
 
     tcp::acceptor acceptor; // Used in boost.asio to take in new clients
     tcp::socket   socket;   // Used in boost.asio to represent clients
-    std::vector<http::handler *> handlers;
+    std::vector<http::handler *> handlers; // Vector fo request handlers
```

Comment:
(1) Just a note: this one isn't a reference because this is the actual data (session should be reference)
(2) You spelled "for" wrong
(3) Maybe comment about how it's data shared by multiple threads